src/hotspot/share/jfr/utilities/jfrBigEndian.hpp

```diff
@@ -26,17 +26,18 @@
 #define SHARE_JFR_UTILITIES_JFRBIGENDIAN_HPP
 
 #include "memory/allStatic.hpp"
-#include "utilities/bytes.hpp"
+#include "utilities/byteswap.hpp"
+#include "utilities/globalDefinitions.hpp"
 #include "utilities/macros.hpp"
 
 #ifndef VM_LITTLE_ENDIAN
 # define bigendian_16(x) (x)
 # define bigendian_32(x) (x)
 # define bigendian_64(x) (x)
 #else
-# define bigendian_16(x) Bytes::swap_u2(x)
-# define bigendian_32(x) Bytes::swap_u4(x)
-# define bigendian_64(x) Bytes::swap_u8(x)
+# define bigendian_16(x) byteswap<u2>(x)
+# define bigendian_32(x) byteswap<u4>(x)
+# define bigendian_64(x) byteswap<u8>(x)
```

Comment:
The explicit template parameters might really be needed, unlike (nearly?) every other call.